types/src/front/api_handlers/public/assistant.ts

```diff
@@ -18,6 +18,13 @@ export const PublicPostMessagesRequestBodySchema = t.intersection([
       fullName: t.union([t.string, t.null]),
       email: t.union([t.string, t.null]),
       profilePictureUrl: t.union([t.string, t.null]),
+      origin: t.union([
```

Comment:
You could have strengthen the type if it were to live in `t.partial` right below.

Reply:
We can't move sub part of objects there can we?

Reply:
You need to move the full origin definition in partial.